tests/test_res_prometheus.c

```diff
@@ -699,6 +702,52 @@ AST_TEST_DEFINE(config_general_core_metrics)
 	return AST_TEST_PASS;
 }
 
+static void safe_bridge_destroy(struct ast_bridge *bridge)
+{
+	if (!bridge) {
+		return;
+	}
+	ast_bridge_destroy(bridge, 0);
```

Comment:
The documentation for `ast_bridge_destroy(...)` indicates that that the bridge argument must be locked on entry.

Reply:
Wouldn't this lead to a deadlock: https://github.com/asterisk/asterisk/blob/869cb0c2608c7602eb372063a28f5160b57ac412/main/bridge.c#L947?

Reply:
Apologies for this. I’m not sure what documentation I was looking at but clearly it was not applicable.